### ISSUE TITLE
Bug: Texture cross on moveable crates

### ISSUE THREAD

**Issue by atombraiderfan** [rr-/Tomb1Main#348]
Textures on moveable crates seem to be divided like a 2x2 image, is there any chance this could be fixed in T1M?

![20220117_171720_Natlas_Mines](https://user-images.githubusercontent.com/94776985/149813619-ed2c0599-64f8-4b3c-986c-249f40ca9f3b.jpg)

**Comment by rr-** [comment]
Not really. The pushblock object has 4 surfaces and each has different UV coords. We'd have to override object data with our own.

**Comment by Richard-L** [comment]
@aredfan turn off bilinear filtering and the problem is gone 😁
